models/maintainer_group.py

```diff
@@ -31,8 +31,8 @@ class MaintainerGroup(Model):
     medium_slug = models.CharField(
         max_length=63,
         blank=True,
+        null = True,
```

Comment:
No need to add `null=True` because this will be a string. Why?

If you add `null=True`, this field can hold both `NULL` and `''` both of them more or less signify the absence of URL in this context. To avoid this redundancy, let's just stick to `blank=True` so that only `""` signifies empty value.

Reply:
There shouldn't be a ` ` (space) before and after `=`.

Reply:
done